design/media_types.go

```diff
@@ -58,6 +58,31 @@ var workItem = MediaType("application/vnd.workitem+json", func() {
 	})
 })
 
+var pagingLinks = Type("pagingLinks", func() {
+	Attribute("prev", String)
+	Attribute("next", String)
+	Attribute("last", String)
+})
+
+// workItemListResponse contains paged results for listing work items and paging links
+var workItemListResponse = MediaType("application/vnd.workitemlist+json", func() {
+	TypeName("WorkItemListResponse")
+	Description("Holds the paginated response to a work item list request")
+	Attribute("links", pagingLinks)
+	Attribute("totalCount", Number)
```

Comment:
Yes, that makes sense. It's even more bogus because the field is not filled in in the request handler (I intended to remove the field, but forgot here).